docs/scripts/buildApi.ts

```diff
@@ -451,6 +451,29 @@ async function buildDocs(options: {
     throw err;
   }
 
+  // Try to get data for the unstyled component
+  try {
```

Comment:
Instead of a try/catch. Would it be enough to try to read the file and if not present ignore it?

Reply:
`readFileSync` is throwing an error if the file doesn't exist, that's why it is necessary so we gotta have the try/catch. Not sure I understand the suggestion

Reply:
In this case, should we reduce the scope down of the try/catch?

Reply:
Done